packages/hardhat/hardhat.config.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I noticed this file is much smaller than the buidler file. This is bound to break some commands in the monorepo package.json. 

Can you comment out the broken commands and add TODOs